### PR TITLE
Feature/style upload modal

### PR DESCRIPTION
**Pull request by katieolson84**
## Description

Edited the styling for the upload case modal. I removed the MUI button and switched it to AntD. I took out the styling and put it into a less file. I moved the svg for the upload icon to the styles folder and imported it in.  I commented out the code for the review form for now. We will need to get this working where the review button only activates when their is a file to review and routes to the 'My Cases' page. 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by MichelleCCodes — APPROVED** [review]
Everything looks good here! Changing the styling library to Ant D is really paying off in cleaning up the code. That's what I've been seeing in your pull requests. Amazing job!

File: src/components/pages/Upload/UploadCase.js [hunk 1]
```diff
@@ -1,98 +1,50 @@
...
 import axios from 'axios';
-import Button from '@material-ui/core/Button';
-import { makeStyles, withStyles } from '@material-ui/core/styles';
-import UploadCaseForm from './UploadCaseForm';
-import CircularProgress from '@material-ui/core/CircularProgress';
-import { notification, Upload, Modal } from 'antd';
-import { CheckCircleOutlined, CloseCircleOutlined } from '@ant-design/icons';
-import { InboxOutlined } from '@ant-design/icons';
+// import UploadCaseForm from './UploadCaseForm';
+import { notification, Upload, Modal, Button, Spin } from 'antd';
```
Comment (paulstgermain) [inline src/components/pages/Upload/UploadCase.js:4]: Wow, the swap to Ant D is doing a lot to clean up our code - nice work!

File: src/components/pages/Upload/UploadCase.js [hunk 2]
```diff
@@ -1,98 +1,50 @@
...
   const [formValueQueue, setFormValueQueue] = useState([]);
-  const classes = useStyles();
   const [isLoading, setIsLoading] = useState(false);
   const [isEditing, setIsEditing] = useState(false);
   const { Dragger } = Upload;
   const [postQueue, setPostQueue] = useState([]);
   const [nextPost, setNextPost] = useState(null);
   const [isModalVisible, setIsModalVisible] = useState(false);
+
+  const spinner = <LoadingOutlined style={{ fontSize: 24 }} spin />;
```
Comment (paulstgermain) [inline src/components/pages/Upload/UploadCase.js:46]: I like the adding of the loading spinner here, little things like this do a lot for user experience!

File: src/components/pages/Upload/UploadCase.js [hunk 3]
```diff
@@ -231,39 +170,59 @@ const UploadCase = ({ authState }) => {
...
+                      <div className="spinner_container">
+                        <Spin indicator={spinner} />
+                      </div>
+                    ) : (
+                      <p />
+                    )}
+                  </>
+                </div>
+              </form>
+            </div>
```
Comment (paulstgermain) [inline src/components/pages/Upload/UploadCase.js:225]: Just wanted to make note of how organized this code is, really nice to see!

**Review by paulstgermain — APPROVED** [review]
Approving these changes on the grounds that all added code is clean and organized, as well as consistent with the changes we've been trying to make. Swapping to Ant D has done a lot for minimizing the amount of code in these files and that's really great to see, and I love the consideration for UX in adding a loading spinner to the modal. Great work!

**Pull request merged**